.github/dependabot.yaml

```diff
@@ -3,20 +3,29 @@ version: 2
 updates:
   - package-ecosystem: "gomod"
     directory: "/"
+    labels:
+      - "area/dependencies"
     schedule:
       interval: "weekly"
       day: "sunday"
       time: "16:00"
     open-pull-requests-limit: 10
     groups:
-      k8s:
+      k8s.io:
         patterns:
           - "k8s.io/api"
           - "k8s.io/apimachinery"
           - "k8s.io/client-go"
+          - "k8s.io/klog/v2"
```

Comment:
wouldn't it make more sense to just use `k8s.io/*`?

Reply:
I did it this way, because I saw that previously it was used like this, but sure we can.

Reply:
Maybe @ramizpolic can comment on the reasons